contracts/tokens/OZStorage.sol

```diff
@@ -2,17 +2,15 @@ pragma solidity ^0.5.0;
 
 contract OZStorage {
 
-    mapping (address => uint256) private _balances;
+    mapping (address => uint256) public _balances;
 
-    mapping (address => mapping (address => uint256)) private _allowed;
+    mapping (address => mapping (address => uint256)) public _allowed;
 
-    uint256 private _totalSupply;
+    uint256 public _totalSupply;
 
-    string private _name;
-    string private _symbol;
-    uint8 private _decimals;
-
-    address private _owner;
+    string public _name;
```

Comment:
I flipped these from internal to public so that etherscan.io would show them. You're correct it could result in a transparency clash but seems a good reason to keep them as "_..." as we shouldn't have any public / external functions with this name schema. If you think this is a bad idea please unresolve and ping me to discuss.

Reply:
Hmm... I agree to show on etherscan we have to compromise something. If we are using those only to show the name/symbol on token tracker then we can do something like SpiceVC token.
[SpiceVC token](https://etherscan.io/address/0x0324dd195d0cd53f9f07bee6a48ee7a20bad738f) is not having the functionality of reading name & symbol from the contract itself using `read contract` but on the etherscan token tracker they are showing the token name/symbol. I think they explicitly add those stuff on the etherscan. May we do the same for our STs instead allowing them to read directly from the contract.

If we are trying to show the creadibility/proof of the contract (whether is the right contract or not) then yes I am agree we can go with your approach.

Reply:
>seems a good reason to keep them as "_..." as we shouldn't have any public / external functions with this name schema.

Clash can still happen. Solidity only stores the function selectors. Function selectors are first 8 digits of the hash of function signatures. There is a relatively high chance of having two completely different functions with the first 8 digits of the hash same. We have the same issue with ST and STGetter, STR and STRGetter (mentioned in my audit :)) but they aren't as major as the function that will be overshadowed in their case will be a getter function. Here, a getter function can overshadow a critical function. It's highly unlikely that this is happening right now or will happen in future when we add more functions but this is a risk I don't think we should be taking. 

I understand that it will be good for etherscan so may I suggest that we add a pre-commit hook/Travis script that will calculate function hashes of all the functions and verify that there are no clashes? Perhaps we can do it in a different PR but it should be part of the 3.0 release. Marking this as unresolved for now.

Reply:
Agree with your approach. A pre-commit scan for any clashing sigs across our non-transparent contracts would be useful. Could probably be widened to look for any sig. clashes across any of our modules (as any sig clash could be problematic if the dApp sent a tx to the wrong contract). I am going to create an Asana / JIRA task for this, then resolve this issue.

Reply:
https://polymath.atlassian.net/browse/SC-1